test-apps/display-test-app/src/backend/DtaElectronMain.ts

```diff
@@ -9,24 +9,25 @@ import { dtaChannel, DtaIpcInterface } from "../common/DtaIpcInterface";
 import { getRpcInterfaces, initializeDtaBackend } from "./Backend";
 import { IpcHandler } from "@bentley/imodeljs-backend";
 
+const mainWindowName = "mainWindow";
 const getWindowSize = () => {
-  let width = 1280;
-  let height = 800;
   const sizeStr = process.env.SVT_WINDOW_SIZE;
   if (typeof sizeStr === "string") {
     const parts = sizeStr.split(",");
     if (parts.length === 2) {
-      const w = Number.parseInt(parts[0], 10);
-      const h = Number.parseInt(parts[1], 10);
+      let width = Number.parseInt(parts[0], 10);
+      let height = Number.parseInt(parts[1], 10);
 
-      if (!Number.isNaN(w))
-        width = w;
+      if (Number.isNaN(width))
+        width = 1280;
 
-      if (!Number.isNaN(h))
-        height = h;
+      if (Number.isNaN(height))
+        height = 1024;
+      return { width, height, x: 100, y: 100 };

```

Comment:
@pmconne Are the hard-coded x, y values OK here? It could be improved if the storage values were read first then overridden by the SVT_WINDOW_SIZE width & height.

Reply:
Looks fine.